src/Components/Components/src/Sections/SectionContent.cs

```diff
@@ -39,33 +40,34 @@ Task IComponent.SetParametersAsync(ParameterView parameters)
     {
         parameters.SetParameterProperties(this);
 
-        if (string.IsNullOrEmpty(Name))
+        if (SectionId is null)
         {
-            throw new InvalidOperationException($"{GetType()} requires a non-empty string parameter '{nameof(Name)}'.");
+            throw new InvalidOperationException($"{GetType()} requires a non-empty string parameter '{nameof(SectionId)}'.");
```

Comment:
Can we update this exception message?
```suggestion
            throw new InvalidOperationException($"{nameof(SectionContent)} requires a non-null value for the parameter '{nameof(SectionId)}'.");
```